src/main/resources/format-pages.txt

```diff
@@ -1678,10 +1678,12 @@ reader = MolecularImagingReader
 [MRC (Medical Research Council)]
 pagename = mrc
 extensions = .mrc, .st, .ali, .map, .rec, .mrcs
-developer = `MRC Laboratory of Molecular Biology <https://www2.mrc-lmb.cam.ac.uk/>`_
+developer = * `MRC Laboratory of Molecular Biology <https://www2.mrc-lmb.cam.ac.uk/>`_ \n
```

Comment:
See https://bio-formats--297.org.readthedocs.build/en/297/formats/mrc.html. Looks like mutliple developers are not rendered correctly. Maybe we should mark `MRC` as `(formerly MRC Laboratory...)` as the ownership/maintenance seems to have been fully transferred to CCP-EM

Reply:
Multiple developers should now be rendered correctly.